Changelog.md

```diff
@@ -10,6 +10,8 @@ Bugfixes:
   * SMTChecker: Fix internal error in magic type access (``block``, ``msg``, ``tx``).
 
 
+
+
```

Comment:
I think this commit can be squashed to the first one.

Reply:
Yea I'm gonna squash everything after review